lib/rules/index.mjs

```diff
@@ -1,256 +1,392 @@
-import { createRequire } from 'node:module';
-// @ts-expect-error -- TS1343: The 'import.meta' meta-property is only allowed when the '--module' option is 'es2020', 'es2022', 'esnext', 'system', 'node16', or 'nodenext'.
-const require = createRequire(import.meta.url);
-
-import importLazy from 'import-lazy';
-
-/** @type {import('stylelint')['rules']} */
+/** @type {import('stylelint').BuiltInRules} */
 const rules = {
```

Comment:
I guess dynamic `import()` with variables may not be a good match with bundlers since they might analyze `import("...")` with string literals. It should be better to write in a straightforward manner.